### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
added a toon of fox channel and corrected some

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -512,18 +512,60 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-filmrisefamily/CDN/1280x720_50000
...
-#EXTINF:-1 tvg-id="FOX28SavannahGAWTGS.us" tvg-name="FOX 28 Savannah GA (WTGS)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/9/99/FOX28WTGS.PNG" group-title="Local",FOX 28 Savannah GA (WTGS) (720p)
-https://content.uplynk.com/channel/e56ba52a1b9d45ad8c8a033fd83fe480.m3u8
 #EXTINF:-1 tvg-id="FoxLifeRussia.us" tvg-name="Fox Life Russia" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Movies",Fox Life Russia (576p)
 http://51.68.140.190:8081/tv/fox-life/playlist.m3u8
 #EXTINF:-1 tvg-id="FoxNews.us" tvg-name="Fox News" tvg-country="US" tvg-language="English" tvg-logo="" group-title="News",Fox News (720p)
 https://bozztv.com/teleyupp/AsEAeOtIxz/playlist.m3u8
 #EXTINF:-1 tvg-id="FoxNewsRadio.us" tvg-name="Fox News Radio" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Z5SATrd.png" group-title="News",Fox News Radio (192p)
 https://fnurtmp-f.akamaihd.net/i/FNRADIO_1@92141/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/qdYfhpZ.jpg" group-title="Mews",Fox News rtmp (720p)
+https://lnc-fox-news-now-rtmp.tubi.video/index.m3u8
```
Comment (freearhey) [inline channels/us.m3u:522]: link is broken

File: channels/us.m3u [hunk 2]
```diff
@@ -512,18 +512,60 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-filmrisefamily/CDN/1280x720_50000
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ktLKla7.png" group-title="Mews",WTTG Fox 5 Washington DC (720p)
+https://lnc-wttg-fox-aws.tubi.video/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Bvkri2L.png" group-title="Mews",WTVT Fox 13 Tampa Bay (720p)
+https://lnc-wtvt-fox-aws.tubi.video/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/fRjmk5r.png" group-title="Mews",WTXF Fox 29 Philadelphia (720p)
+https://lnc-wtxf-fox-aws.tubi.video/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/X491SzR.gif" group-title="Sport",Fox Sports pluto (720p)
+https://stitcher.pluto.tv/stitch/hls/channel/5a74b8e1e22a61737979c6bf/master.m3u8?deviceType=web&deviceMake=Chrome&deviceModel=Chrome&sid=3fab7580-8b86-11e8-a44b-996a399dacd8&deviceId=3fab0050-8b86-11e8-a44b-996a399dacd8&deviceVersion=67.0.3396.99&appVersion=2.0.0&deviceDNT=0&userId=&advertisingId=&deviceLat=38.8177&deviceLon=-77.1527&app_name=&appName=&appStoreUrl=&serverSideAds=true
```
Comment (freearhey) [inline channels/us.m3u:568]: Link to Pluto TV's "Anime All Ages" channel

File: channels/us.m3u [hunk 3]
```diff
@@ -512,18 +512,60 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-filmrisefamily/CDN/1280x720_50000
...
-https://content.uplynk.com/channel/e56ba52a1b9d45ad8c8a033fd83fe480.m3u8
 #EXTINF:-1 tvg-id="FoxLifeRussia.us" tvg-name="Fox Life Russia" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Movies",Fox Life Russia (576p)
 http://51.68.140.190:8081/tv/fox-life/playlist.m3u8
 #EXTINF:-1 tvg-id="FoxNews.us" tvg-name="Fox News" tvg-country="US" tvg-language="English" tvg-logo="" group-title="News",Fox News (720p)
 https://bozztv.com/teleyupp/AsEAeOtIxz/playlist.m3u8
 #EXTINF:-1 tvg-id="FoxNewsRadio.us" tvg-name="Fox News Radio" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Z5SATrd.png" group-title="News",Fox News Radio (192p)
 https://fnurtmp-f.akamaihd.net/i/FNRADIO_1@92141/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/qdYfhpZ.jpg" group-title="Mews",Fox News rtmp (720p)
+https://lnc-fox-news-now-rtmp.tubi.video/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/qdYfhpZ.jpg" group-title="Mews",Fox News Now (720p)
```
Comment (freearhey) [inline channels/us.m3u:523]: Mistake in the group-title

**Review by freearhey — CHANGES_REQUESTED** [review]
Please correct all marked links so we can approve this PR

File: channels/us.m3u (hunk 2 above)
Comment (Sphinxroot) [reply channels/us.m3u:566]: a bit strange for pluto  i get sport from it

File: channels/us.m3u (hunk 1 above)
Comment (Sphinxroot) [reply channels/us.m3u]: ok removed for now

File: channels/us.m3u (hunk 3 above)
Comment (Sphinxroot) [reply channels/us.m3u]: corrected   sorry  big finger on the kewbord

File: channels/us.m3u (hunk 2 above)
Comment (Sphinxroot) [reply channels/us.m3u:566]: https://i.imgur.com/cooV6hr.png

**Review by freearhey — APPROVED** [review]

**Pull request merged**